R/learner_abess_classif_abess.R

```diff
@@ -0,0 +1,155 @@
+#' "scikit-survival: A Library for Time-to-Event Analysis Built on Top of scikit-learn."
+#' J. Mach. Learn. Res., 21(212), 1-6.
+#'
+#' Yanhang Zhang, Junxian Zhu, Jin Zhu, and Xueqin Wang (2021).
+#' "Certifiably Polynomial Algorithm for Best Group Subset Selection."
+#' arXiv preprint arXiv:2104.12576.
+#'
+#' Qiang Sun and Heping Zhang (2020).
+#' "Targeted Inference Involving High-Dimensional Data Using Nuisance Penalized Regression."
+#' Journal of the American Statistical Association.
+#' \doi{10.1080/01621459.2020.1737079}
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+LearnerClassifAbess = R6Class("LearnerClassifAbess",
+  inherit = LearnerClassif,
+
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      ps = ps(
+        family = p_fct(
+          levels = c("binomial", "multinomial"),
+          default = "binomial", tags = "train"),
+
+        tune.path = p_fct(
+          levels = c("sequence", "gsection"),
+          default = "sequence", tags = "train"),
+        tune.type = p_fct(
+          levels = c("gic", "ebic", "bic", "aic", "cv"),
+          default = "gic", tags = "train"),
+        weight = p_uty(default = NULL, tags = "train"),
+        normalize = p_int(lower = 0L, upper = 3L, default = 2L, tags = "train"),
+        c.max = p_int(lower = 1L, upper = Inf, default = 2L, tags = "train"),
+        support.size = p_uty(tags = c("train", "predict")),
+        lambda = p_dbl(lower = 0, upper = Inf, default = 0, tags = "train"),
+        always.include = p_uty(default = NULL, tags = "train"),
+        group.index = p_uty(default = NULL, tags = "train"),
+        splicing.type = p_int(lower = 1L, upper = 2L, default = 2, tags = "train"),
+        max.splicing.iter = p_int(
+          lower = 1L, upper = Inf,
+          default = 20L, tags = "train"),
+        screening.num = p_int(default = 2, tags = "train"),
+        important.search = p_int(default = 128, tags = "train"),
+        warm.start = p_lgl(default = TRUE, tags = "train"),
+        nfolds = p_int(lower = 1L, upper = Inf, default = 5L, tags = "train"),
+        newton = p_fct(levels = c("exact", "approx"), tags = "train"),
+        newton.thresh = p_dbl(default = 1e-6, tags = "train"),
+        max.newton.iter = p_int(
+          lower = 1L, upper = Inf,
+          default = 60, tags = "train"),
+        early.stop = p_lgl(default = FALSE, tags = "train"),
+        num.threads = p_int(lower = 0L, upper = Inf, default = 0L, tags = "train"),
+        seed = p_int(lower = 0L, default = 1L, tags = "train"),
+        subset = p_uty(tags = "train", custom_check = check_string),
+        na.action = p_uty(default = stats::na.omit, tags = "train")
+      )
+      ps$add_dep(id = "support.size", on = "tune.path", CondEqual$new("sequence"))
+      ps$add_dep(id = "num.threads", on = "tune.type", CondEqual$new("cv"))
+      ps$values$newton = "approx"
```

Comment:
You have changed or set a default but haven't written this in the documentation, please add above https://mlr3book.mlr-org.com/extending.html#param-set

Reply:
thank you for your detailed reply, but I am not sure where the "learner documentation" is?